src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/request/LicenceCaseloadSearchRequest.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.request
+
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotNull
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.ProbationSearchSortByRequest
+
+@Schema(description = "Request object for searching for offenders within a set of teams attached to a staff member")
+data class LicenceCaseloadSearchRequest(
```

Comment:
is this only used by the client? if so it should probably live inside the service/probation folder. 

This model package is used for representations that are consumed/produced by our API

Reply:
think the same for `ProbationSearchSortByRequest`